github/github_test.go

```diff
@@ -169,6 +169,19 @@ func testJSONMarshal(t *testing.T, v interface{}, want string) {
 	}
 }
 
+func testAddOptions(t *testing.T, url string, v interface{}, want string) {
```

Comment:
Since the structures passed in must have "url" tags in the fields (more in [addOptions](https://github.com/google/go-github/blob/master/github/github.go#L286)), is it valid to test if the fields have this tag?

Something like this:

```go
st := reflect.TypeOf(v)
for i := 0; i < st.NumField(); i++ {
	field := st.Field(i)
	if alias, ok := field.Tag.Lookup("url"); ok {
		if alias == "" {
			// t.Errorf("...")
		}
	} else {
		// t.Errorf("...")
	}
}
```

Reply:
Sure.  Please add a comment before line 172 that describes this requirement so that people will not start adding `testAddOptions` to every test.

In fact, let's make the function name more descriptive.  How about `testAddURLOptions` ?

Reply:
Or maybe `testAdURLTags` ?